src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -422,15 +436,16 @@ Namespace Microsoft.CodeAnalysis.Semantics
                             _semanticModel,
                             syntax,
                             type:=Nothing,
-                            constantValue:=Nothing)
+                            constantValue:=Nothing,
+                            isImplicit:=Nothing) ' TODO
```

Comment:
>' TODO [](start = 49, length = 6)

The "TODO" comment should either be removed or replaced with a ```PROTOTYPE(<feature>)``` comment. Also, it is better to not use ```Nothing``` literal for boolean values.